src/components/Registration/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React,{useState} from 'react'
+import userIco from './../Registration/user_icon.png';
+
+const Registration = () => {
+
+  const [user,setUser] = useState(
+    {
+      username: '',
+      email: '',
+      password: '',
+      passwordConfirm: '',
+    }
+ )
+
+ const handleSubmit = (e) => {
+  e.preventDefault() 
+  if (user.password === user.passwordConfirm && user.username !== '' && user.email !== '' && user.password !== '' && user.passwordConfirm !== '') {
```

Comment:
kdybys to chtěla zkrátit, dá se udělat i něco jako:
```if([user.username, user.email].every((value) => value !== "") ```
;)
reálně bys na takovéhle věci použila pak nějakou validační knihovnu protože dělat to ručně je vopruz https://www.cssscript.com/best-javascript-form-validator/ (a nebo jen html 5 validace https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation)